generate_testdata.sh

```diff
@@ -18,7 +18,7 @@ source common.sh
 
 build_kb() {
   header_text "Building kubebuilder"
-  go build -o ./bin/kubebuilder ./cmd
+  go build -o ./bin/kubebuilder sigs.k8s.io/kubebuilder/v3/cmd
```

Comment:
I don't think this is necessary

Reply:
This is definetely not needed and the `./cmd` will be valid forever.

Reply:
Solved.